modules/AOR_Reports/AOR_Report.php

```diff
@@ -692,8 +692,18 @@ function build_report_html($offset = -1, $links = true, $group_value = '', $tabl
             foreach($fields as $name => $att){
                 if($att['display']){
                     $html .= "<td class='' valign='top' align='left'>";
-                    if($att['link'] && $links){
-                        $html .= "<a href='" . $sugar_config['site_url'] . "/index.php?module=".$att['module']."&action=DetailView&record=".$row[$att['alias'].'_id']."'>";
+                    if(!empty($att['link']) && $links){
+                        if(
+                            !empty(array_key_exists($att['field'], $field_bean->field_name_map)) AND
```

Comment:
I also get that write context error: PHP Fatal error:  Can't use function return value in write context.
Change the function:
array_key_exists($att['field'], $field_bean->field_name_map) !== FALSE